tests/e2e/pageObjects/loginPage.js

```diff
@@ -0,0 +1,35 @@
+export class LoginPage {
+
+    //Elements
+    welcomeText = "div.login-header";
+    emailField = "input[name='email']";
+    passwordField = "input[name='password']";
+    loginButton = "span[data-reactid='.0.1.0.1.2:$0.0.0.1.0']";
+    signUpButton = "div[data-reactid='.0.1.0.2.1']";
```

Comment:
Are you sure? Provided by you selector describes the whole "footer" of the menu, but I need only Sign Up button here.